OnlineShopUI/online-shop-ui/src/components/SideMenuList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './SideMenuList.scss';
+
+const SideMenuList = (props) => {
+  return (
+    <div className="categories__menu">
+      <p>
+        <strong>{props.nameCategory}</strong>
+      </p>
+      <ul>
+        <li>
+          <a href="#">{props.item}</a>
+        </li>
+        <li>
+          <a href="#">{props.item1}</a>
+        </li>
+        <li>
+          <a href="#">{props.item2}</a>
+        </li>
+        <li>
+          <a href="#">{props.item3}</a>
+        </li>
+        <li>
+          <a href="#">{props.item4}</a>
+        </li>
```

Comment:
We are able to use loop for it, and that's why I recommened to start count from 1 instead of nothing

Reply:
Use map here to generate tegs